test/unit/background/accounts-test.js

```diff
@@ -103,24 +94,27 @@ describe("background > accounts", () => {
   });
 
   describe("Account", () => {
-    let acct;
+    let acct, storage;
     const unauthedInfo = {
       uid: "1234",
       access_token: "KhDtmS0a98vx6fe0HB0XhrtXEuYtB6nDF6aC-rwbufnYvQDgTnvxzZlFyHjB5fcF95AGi2TysUUyXBbprHIQ9g",
       id_token: "eyJhbGciOiJSUzI1NiIsImtpZCI6IjBDWTBJb3RVVHBtWDFDbXJqMWNwcTB6aFBkd1NtalJSaWlJNzduMmMtMFkifQ.eyJ1aWQiOiIxMjM0IiwiaXNzIjoic2NvcGVkLWtleXMifQ.pX8I1LqCD849Gp0TzKcS6LM_fko0gc7wkSzBgPaxFDyF8AZrWn9-HTRoW-9YIuHujLzldbI1k34VeSHNM85vkPjm_AxbBKuXEiVJQdcCAxNjbSQmM1dOX6kZKwN4oDu8X4BB3CwQq5eXioYYiPur149O_I2bhFDuMBtQBoQosZtOScuKliXcURuWEwhYcnHe8axit0fQ0vd1FOJK3300hccqcZNoHGXrSVj42mdo_aSREOcwSUP4i0r0aCfJqnxai43uy1C5l54mSN1KzqGeasx60lWPU-Jm3gPm_2CXRWbfWxF3-OnxhMhSQiS90kefX81H03ZYVShDutsx55d0tQ",
+      expires_at: Math.floor(Date.now() / 1000) + 1209600,
```

Comment:
Not sure if we want to add a comment (or `const`) about the seemingly magic "1,209,600" number represents (my best guess was 2 weeks in seconds).

Reply:
@pdehaan for testing, the value is really arbitrary.  Personally, I'm inclined to let it be arbitrary for now.